### ISSUE TITLE
SCTE management and support 

### ISSUE THREAD

**Issue by ffiore1** [awslabs/video-on-demand-on-aws#57]
Hi, would need to add Scte markers for advertising.

Apart from using a template having already embedded the esam sccml file, is there any other way using the solution? Is the management of an external xml file for that already in your roadmap?

**Comment by dscpinheiro** [comment]
Hi @ffiore1,

No, this is not possible without using custom templates. I'll add this feature request to our backlog.

**Comment by ffiore1** [comment]
just to clarify your reply, are you suggesting that using a custom template is already possible to insert SCTE? if that is the case, which is the syntax for that?

@dscpinheiro 
Negatively, just to clarify possible ideal requirements, it would be great if it is going to be possible insert in the custom template the reference to internal (or external file) xml which would be placed in the same template location.

**Comment by dscpinheiro** [comment]
> just to clarify your reply, are you suggesting that using a custom template is already possible to insert SCTE? if that is the case, which is the syntax for that?

Yes, since you can specify the SCTE settings in a job, you could use a job template. The request format is described on this page: https://docs.aws.amazon.com/mediaconvert/latest/apireference/jobtemplates.html

**Comment by aassadza** [comment]
Addressed with version v5.1.0

**Comment by ffiore1** [comment]
@aassadza  could you please provide any details? no reference to it in the changelog.

additional question: is tere an upgrade/update process for switching from 5.0 to 5.10?

**Comment by ffiore1** [comment]
It is still not clear how you have implemented that in the 5.10 release. Using that, have you noticed any bugs with large files?

I have modified the encode lambda function of your stack to submit a job with proper esam, using an additional csv with the cuepoints information specified in the triggering job template.

what we have noticed is that hls works well but not dash. it would seem there is a bug in MediaConvert  managing ESAM with large files. Didn't you experience that?  any alternative strategy?
